less/index.less

```diff
@@ -57,3 +57,175 @@ html, body {
     height: 100%;
 }
 
+@primary: #203731;
+@secondary: #FFB612;
+.flex {
+	display: flex;
+	}
+
+.h2resize {
+	font-size: 25px;
+}
+
+.center {
+	margin: 0 auto;
+}
+
+.container {
+	background-color: @primary;
+	color: @secondary;
+
+	nav {
+		.flex;
+		flex-wrap: nowrap;
+		justify-content: space-around;
+		margin-bottom: 100px;
+		max-width: 1000px;
+		width: 100%;
+		.center;
+		background-color: @secondary;
+
+		a {
+			font-size: 20px;
+			text-align: center;
+			width: 15%;
+			text-decoration: none;
+			color: @primary;
+			border: 2px solid @secondary;
+			padding: 0px 15px;
+		}
+
+		@media (max-width: 500px){
+				flex-wrap: wrap;
+				flex-direction: column;
+				
+				a{
+					width: 100%;
+					border-bottom: 2px solid @primary;
+				}
+				}
+	}
+
+	h1{
+		font-size: 50px;
+		text-align: center;
+		margin-bottom: 100px;
+	}
+
+	.content {
+		.intro {
+			.flex;
+			align-items: left;
+			width: 85%;
+			flex-direction: row;
+			margin-left: 5%;
+			margin-bottom: 30px;
+
+			h2{
+				.h2resize;
+				margin-bottom: 15px;
+			}
+
+			p{
+				font-size: 20px;
+				width: 60%;
+			}
+			
+			img {
+				width: 45%;
+			}
+
+			@media (max-width: 500px){
```

Comment:
great job demonstrating your master of nesting and nesting media queries. It shows you fully understand how to execute on the lesson and expectations.